### DISCUSSION TITLE
Direction of input force and moment in stress-strain recovery

### DISCUSSION THREAD

**Discussion by vacupenda** [wenbinyugroup/.github#31] [VABS Related Discussions]
I would like to know the directions of the six components of the input loads (F1, F2, F3, M1, M2, M3) required for stress-strain recovery for the airfoil, and whether they are in the following positions?
<global measure="stress">
  <displacements> 0 0 0 </displacements>
  <rotations> 1 0 0 0 1 0 0 0 1 </rotations>
  <loads> F1 F2 F3 M1 M2 M3</loads>
</global>
![mmexport1731387468427](https://github.com/user-attachments/assets/ff14ffda-1e72-467e-a0ba-a5c8db90c5fa)

**Comment by wenbinyu-PU** [comment]
They are arranged as you stated. Their directions align with the beam coordinate system you used for beam analysis.

**Reply by vacupenda** [reply]
Got it, thank you. So, if I select according to the following coordinate system, F1 should be along the X1 direction, and so on, right?
![Screenshot_20241112_224910](https://github.com/user-attachments/assets/a4b1e5bf-2cae-48e3-9db9-0c001de7e322)

**Reply by wenbinyu-PU** [reply]
Yes.
